cucumber/api/features/step_definitions/policy_load_steps.rb

```diff
@@ -0,0 +1,69 @@
+require 'English'
+
+When(/^I load a large policy with POST$/) do
+  # Generate a large policy with 1000 variables
+  policy_body = "- &my-variables\n" + [*1..1000].map do |ndx|
+    %Q(
+      - !variable
+        id: obj_Safe_00c_0_#{ndx}/password
+        annotations:
+          cyberark-vault: 'true'
+          cyberark-vault/accounts: AutomationVault/Safe_00c/obj_Safe_00c_0_10
+    )
+  end.join("\n")
+
+  path = '/policies/cucumber/policy/dev/db'
+
+  try_request true do
+    post_json path, policy_body
+  end
+end
+
+When('I use curl to load a policy with special characters and no content type') do
+  policy_body = %Q(
+    - &active-variables
+      - !variable
+        id: AccountWithSpecialCharacters~!@#$%^&(){}[]-+=,Name/password
+        annotations:
+          cyberark-vault: 'true'
+          cyberark-vault/accounts: AutomationVault/safeToSync/AccountWithSpecialCharacters~!@#$%^&(){}[]-+=,Name
+      - !variable
+        id: AccountWithSpecialCharacters~!@#$%^&(){}[]-+=,Name/username
+        annotations:
+          cyberark-vault: 'true'
+          cyberark-vault/accounts: AutomationVault/safeToSync/AccountWithSpecialCharacters~!@#$%^&(){}[]-+=,Name
+      - !variable
```

Comment:
These other two variables aren't really doing anything differently, right? They all use the same set of special characters. I think we only need the one use case here unless I'm missing something.

Reply:
Yes, you're right. I was able to simplify both sets of policies to a more simplified version that still caused the failure.